gateway/api/rest/pom.xml

```diff
@@ -33,17 +33,6 @@
       <artifactId>swagger-annotations</artifactId>
     </dependency>
     <dependency>
-      <groupId>io.swagger</groupId>
```

Comment:
Actually I just did a little bit more sleuthing and I think this change is OK.  The relevant commit is here:

https://github.com/apiman/apiman/commit/51fbb51c325ecfd0068853f3d516464d5cd2e8c1

You added swagger annotations to the API as well as a swagger 2.0 resource to the jax-rs layer.  The latter requires swagger-jaxrs to be added to the `gateway/platforms/war/micro/pom.xml` file, which you did.  So I think *this* reference can safely be removed, since the `gateway/api/rest/` module really only needs the swagger-annotations dependency.